py_jama_rest_client/client.py

```diff
@@ -205,6 +205,21 @@ def put_item_lock(self, item_id, locked):
         response = self.__core.put(resource_path, data=json.dumps(body), headers=headers)
         return self.__handle_response_status(response)
 
+    def get_item_tags(self, item_id):
+        """
+        Return all tags for the item with the specified ID
+
+        Args:
+            item_id: the item id of the item to fetch
+
+        Returns: a dictionary object representing the item's tags
+
+        """
+        resource_path = 'items/' + str(item_id) + '/tags'
+        response = self.__core.get(resource_path)
```

Comment:
This endpoint can potentially return multiple pages.  We will need to call the get_all method to return all the available pages of data.

Reply:
Thanks for the feedback. Please see the updated code.